### PR TITLE
Make tests runnable locally without errors or noise.

### PR DESCRIPTION
**Pull request by gitlost**
Fixes https://github.com/wp-cli/wp-cli/issues/4210

Adds "Given an empty XXX directory" for use by `cli-info.feature` to clear `PACKAGE_PATH` so it's re-runnable.

Fixes `UPDATE_VERSION` in `cli.feature` to get the correct latest version.

Suppresses PHP notices in `framework.feature` thrown by `wp-db.php`.

(Also needs https://github.com/wp-cli/wp-cli/pull/4208 to remove noise completely.)

### CODE REVIEW

File: features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -8,6 +8,9 @@
 use \WP_CLI\Process;
 use \WP_CLI\Utils;
 
+use Symfony\Component\Filesystem\Filesystem,
+	Symfony\Component\Filesystem\Exception\IOExceptionInterface;
```
Comment (danielbachhuber) [inline features/bootstrap/FeatureContext.php]: We should remove the Symfony dependency, see https://github.com/wp-cli/wp-cli/pull/4209#issuecomment-314093447

Comment (danielbachhuber) [inline features/bootstrap/FeatureContext.php]: I've done so in 5d4acf18

**Comment by gitlost** [comment]
I think it would be nicer to keep `remove_dir()` in `FeatureContext`, and for all dir deletes to use it, eg
```php
Process::create( Utils\esc_cmd( 'rm -rf %s', $dir ) )->run_check();
```
rather than having variously varying variants around.

**Comment by danielbachhuber** [comment]
@gitlost Ok. Do you want to do that in this PR or another?

**Comment by gitlost** [comment]
I'll do it in another PR and let this proceed...

**Pull request merged**